gframe/deck_manager.h

```diff
@@ -1,11 +1,16 @@
 #ifndef DECKMANAGER_H
 #define DECKMANAGER_H
 
-#include "config.h"
```

Comment:
We should always include global config. BTW, this is a big change but not related to this pull request.

Reply:
The header file does not need config.h
But I agree that it should be put in another PR.